packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.js

```diff
@@ -19,7 +23,8 @@ const ItemsReview = props => {
     const talonProps = useItemsReview({
         queries: {
             getItemsInCart: LIST_OF_PRODUCTS_IN_CART_QUERY
-        }
+        },
+        data: props.data
```

Comment:
I want to re-use this component's UI and other logic but not the queries. I'm open to other approaches but this one seemed the most direct. If data is passed to the talon it will use it instead of making a query.